style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
[OPTIONAL] I noticed these two lines of code have been repeated at lines #17 and #18 of the style.css file.
Let's try to follow the [DRY](https://openweb.eu.org/articles/dry-approaches-don-t-repeat-yourself) method.

You could do something like this:
```
.center-items {
    justify-content: center;
    align-items: center;
}
```
And use that class inside your HTML element like this:
```
<div class="content center-items"></div>
```